devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -101,5 +142,172 @@ protected SpanByte SliceGenericBuffer(int start, int length)
+
+            if (useWhite)
+                _genericBuffer[idx] |= (byte)(1 << (y & 7));
+            else
+                _genericBuffer[idx] &= (byte)~(1 << (y & 7));
+
+            //INVERSE COLOR: _genericBuffer[idx] ^= (byte)(1 << (y & 7));
+        }
+
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="x"></param>
+        /// <param name="y"></param>
+        /// <param name="width"></param>
+        /// <param name="height"></param>
+        /// <param name="bitmap"></param>
+        public void DrawBitmap(int x, int y, int width, int height, byte[] bitmap)
+        {
+            if ((width * height) != bitmap.Length)
+                throw new ArgumentException("Width and height do not match the bitmap size.");
+
+            for (var yO = 0; yO < height; yO++)
+            {
+                for (var xA = 0; xA < width; xA++)
+                {
+                    var b = bitmap[(yO * width) + xA];
+                    byte mask = 0x01;
+                    for (var pixel = 0; pixel < 8; pixel++)
+                    {
+                        DrawPixel(x + (8 * xA) + pixel, y + yO, (b & mask) > 0);
+                        mask <<= 1;
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="x"></param>
+        /// <param name="y"></param>
+        /// <param name="str"></param>
+        public void DrawString(int x, int y, string str)
+        {
+            byte[] bitMap = GetTextBytes(str);
+
+            DrawBitmap(x, y, bitMap.Length / Font.Height, Font.Height, bitMap);
+        }
+
+
+        /// <summary>
+        /// Get the bytes to be drawn on the screen for text, from the font
+        /// </summary>
+        /// <param name="text">Strint to be shown on the screen.</param>
+        /// <returns></returns>
+        byte[] GetTextBytes(string text)
+        {
+            byte[] bitMap;
+
+            if (Font.Width == 8)
+            {
+                bitMap = new byte[text.Length * Font.Height * Font.Width / 8];
+
+                for (int i = 0; i < text.Length; i++)
+                {
+                    var characterMap = Font[text[i]];
+
+                    for (int segment = 0; segment < Font.Height; segment++)
+                    {
+                        bitMap[i + (segment * text.Length)] = characterMap[segment];
+                    }
+                }
+            }
+            else
+            {
+                throw new Exception("Font width must be 8");
+            }
+
+            return bitMap;
+        }
+
+        /// <summary>
+        /// Displays the information on the screen.
+        /// </summary>
+        public void Display()
+        {
+            for (byte i = 0; i < Pages; i++)
+            {
+                _pageCmd[1] = (byte)(PageAddress.Page0 + i); // page number
+                _i2cDevice.Write(_pageCmd);
```

Comment:
See the comment in the PR, seems like this part may need to be adjusted for some screens